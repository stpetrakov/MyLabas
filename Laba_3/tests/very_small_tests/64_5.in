32330 32268 25713 304 30813 13071 18681 293 18199 21845 18446 7795 15042 17871 2413 16951 14106 15389 32194 5046 6748 24915 7779 2372 6817 2046 21822 101 3434 9570 8465 8497 19836 28678 18905 4349 13038 18291 14732 14391 20864 18744 19722 8082 28974 14129 12625 10339 29957 4873 8433 9291 9270 4862 5569 4119 2409 21476 17103 30180 23635 27757 27293 14983 