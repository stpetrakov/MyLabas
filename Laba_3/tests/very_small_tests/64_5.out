101 293 304 2046 2372 2409 2413 3434 4119 4349 4862 4873 5046 5569 6748 6817 7779 7795 8082 8433 8465 8497 9270 9291 9570 10339 12625 13038 13071 14106 14129 14391 14732 14983 15042 15389 16951 17103 17871 18199 18291 18446 18681 18744 18905 19722 19836 20864 21476 21822 21845 23635 24915 25713 27293 27757 28678 28974 29957 30180 30813 32194 32268 32330 